### PR TITLE
Very poor attempt at understanding this

### PR DESCRIPTION
**Pull request by psikoi**

### CODE REVIEW

**Comment by psikoi** [comment]
@deathbeam pls

File: src/components/home.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+.feature .card-title a {
+  color: white;
+}
+
+.feature .card-text {
+  color: #a4a4a4;
+  font-size: 10pt;
+}
+
+.section-link {
```
Comment (deathbeam) [inline src/components/home.css]: you can just specify the color here (instead of defining it in home component)

File: src/routes/home.js [hunk 2]
```diff
@@ -42,17 +42,25 @@ class Home extends Component {
...
-            <Link href="/features" style={{ fontSize: 18 }}>
-              See all features...
-            </Link>
-          </h1>
-          <hr />
-          <div class="row">
-            {features.filter(feature => feature.home).map(feature => (
-              <Feature key={feature.title} {...feature} />
-            ))}
+          <div style={{ background: '#171717' }}>
```
Comment (deathbeam) [inline src/routes/home.js]: Maybe just make features-holder CSS (or something) and set the background there or if possible set it directly in .features class

File: src/components/layout.css [hunk 3]
```diff
@@ -1,4 +1,5 @@
 #layout {
-  padding-top: 60px;
-  padding-bottom: 15px;
+  padding: 0px;
```
Comment (deathbeam) [inline src/components/layout.css]: This will most likely break rest of the website pages that depend on this

File: src/components/home.css [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  font-size: 12pt;
+  float: right;
```
Comment (deathbeam) [inline src/components/home.css]: Just use https://getbootstrap.com/docs/4.0/utilities/float/

File: src/components/home.css [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  margin: auto;
+}
+
+.features h1 {
+  color: white;
+  font-size: 30pt;
+  margin-bottom: 35px;
+}
+
+.feature {
```
Comment (deathbeam) [inline src/components/home.css]: all these .feature classes should be in `components/feature.css` and `components/feature.js` should import it

File: src/components/home.css [hunk 6]
```diff
@@ -0,0 +1,42 @@
+.section {
```
Comment (deathbeam) [inline src/components/home.css]: this looks unused

File: src/components/app.js [hunk 7]
```diff
@@ -4,6 +4,7 @@ import { h } from 'preact'
 import { connect } from 'preact-redux'
 import Router from 'preact-router'
 import './app.css'
+import './home.css'
```
Comment (deathbeam) [inline src/components/app.js]: import this in `routes/home.js`

File: src/components/home.css [hunk 8]
```diff
@@ -0,0 +1,32 @@
+.home-layout {
```
Comment (deathbeam) [inline src/components/home.css]: this file should be located next to whatever is using it. e.g it should be in `routes/home.css`.

File: src/routes/home.js [hunk 9]
```diff
@@ -41,18 +42,21 @@ class Home extends Component {
...
-            <Link href="/features" style={{ fontSize: 18 }}>
-              See all features...
-            </Link>
-          </h1>
-          <hr />
-          <div class="row">
-            {features.filter(feature => feature.home).map(feature => (
-              <Feature key={feature.title} {...feature} />
-            ))}
+        <Layout fullWidth class="home-layout">
```
Comment (deathbeam) [inline src/routes/home.js]: As we talked on Discord, probably moving the features section outside of <Layout> would be cleaner solution

File: src/components/home.css [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+  overflow-x: hidden;
+}
+
+.home-section-container {
+  background: #171717;
+  padding: 50px 0px;
+}
+
+.home-section {
+  max-width: 960px;
```
Comment (deathbeam) [inline src/components/home.css]: Avoid doing this, use `container` class whenever you need to specify the max-width

**Pull request closed without merging**